graph-data.rs/Cargo.toml

```
@@ -13,3 +13,5 @@ anyhow = "1.0"
 regex = "^1.1.0"
 semver = { version = "^0.9.0", features = [ "serde" ] }
 toml = "^0.4.10"
+protoc = "=2.8.0"
```

I think we could still allow patch release updates

```suggestion
protoc = "^2.8.0"
```

---

Not sure why, but it makes `cargo generate-lockfiles` bring in the cursed 2.18.0. Tilde notation makes it update to `2.8.2`, which is what we want

---

Ah, yes, caret notation would fix the major version only - `^1.2.3  :=  >=1.2.3, <2.0.0`, so `^2.8.0 := >=2.8.0, < 3.0.0`, which is undesirable